### DISCUSSION TITLE
Обсуждаем идеи, вопросы и все, что угодно

### DISCUSSION THREAD

**Discussion by BlizD** [BlizD/Tasks#270] [Общее]
## 👋 Всем привет.
Обсуждаем идеи, вопросы и все, что угодно.
Здесь можно обсудить, задать вопрос, поделится идеей без открытия задачи.

**Comment by nonameformr** [comment]
Возможно ли сделать эту систему в виде расширения?

**Reply by BlizD** [reply]
> Возможно ли сделать эту систему в виде расширения?

Добрый день.
Нет, т.к. используются механизмы БСП, а расширения пока не умеют изменять определяемые типы, использовать регламентные задания и т.п. 
Плюс в каждой конфигурации используется своя версия БСП и это может быть проблемой для расширения.
Переделка на расширения будет очень трудоемким и длительным процессом.

**Comment by IuriiChernomorov** [comment]
Приветствую, почему в "узПланированиеПроекта" в форме "ФормаЗадачиУправляемая" закомментирована передача параметров в форму владельца при закрытии? И остались какие-то рудименты на форме владельца, в частности интересует "ЗаполнитьПроектПоЗадачамИБ". Вроде функционал дельный, не пойму почему вырезано - оно сырое и надо допилить или функционал срулил в какую-то другую сторону? Спасибо!

**Reply by BlizD** [reply]
@IuriiChernomorov, добрый день.

>Приветствую, почему в "узПланированиеПроекта" в форме "ФормаЗадачиУправляемая" закомментирована передача параметров в форму владельца при закрытии? 

А что там передается? 
Возможно было закомментировано, чтобы просто не выдавало ошибку.

>И остались какие-то рудименты на форме владельца, в частности интересует "ЗаполнитьПроектПоЗадачамИБ". Вроде функционал дельный, не пойму почему вырезано - оно сырое и надо допилить или функционал срулил в какую-то другую сторону?

Пока была просто цель внедрить уже существующую обработку "Планирование проекта" от другого разработчика.
А далее посмотреть будет ли востребован функционал и будут ли желающие его доработать.

подробнее об обработке и авторе:
>https://github.com/BlizD/Tasks/issues/222 Встроена обработка для планирования проекта с разрешения автора (Сергей Головченко(rare-avis) https://infostart.ru/public/100480/)
обработка, аналог MS Project в 1Се для работы с задачами: параллельный ввод данных по задачам и график Ганта, дополнительный режим в виде календаря (планировщик), связывание задач и пересчет длительности и дат с учетом выходных, видов связей и запаздываний, расчет затрат, сохранение/открытие проектов в файл на диск или файл ИБ, импорт/экспорт документов Microsoft Project.

Один из пользователей нашей конфигурации прислал краткую [инструкцию ](https://github.com/BlizD/Tasks/wiki/%D0%9A%D0%B5%D0%B9%D1%81-%D0%BA%D0%B0%D0%BA-%D0%BC%D0%BE%D0%B6%D0%BD%D0%BE-%D0%B2%D0%B5%D1%81%D1%82%D0%B8-%D0%B8-%D0%BF%D0%BB%D0%B0%D0%BD%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82-%D0%B2-%D0%BA%D0%BE%D0%BD%D1%84%D0%B8%D0%B3%D1%83%D1%80%D0%B0%D1%86%D0%B8%D0%B8)как можно вести проект через эту обработку.

После первичного внедрения обработки, был создана следующая задача https://github.com/BlizD/Tasks/issues/340, на которую ищем добровольца-разработчика, кому она в первую очередь нужна.
